ksql-engine/src/test/java/io/confluent/ksql/services/DefaultConnectClientTest.java

```diff
@@ -66,7 +67,7 @@
 
   @Before
   public void setup() {
-    client = new DefaultConnectClient("http://localhost:" + wireMockRule.port());
+    client = new DefaultConnectClient("http://localhost:" + wireMockRule.port(), Optional.empty());
```

Comment:
Missing test? Feels like this test should be testing that headers are set when making calls...